### PR TITLE
feat: Add k8s glue agent deployment

### PR DESCRIPTION
**Pull request by nielstenboom**
This PR adds the ClearML k8s glue agent implementation. We use this to deploy new experiments as pods in our k8s cluster combined with a GPU autoscaling group, which allows for swiftly scaling our training capacity if needed!

Please let me know what you think of it!

### CODE REVIEW

File: charts/clearml/values.yaml [hunk 1]
```diff
@@ -291,6 +291,27 @@ agentGroups:
 
     affinity: {}
 
+# This agent will spawn queued experiments in new pods, a good use case is to combine this with
+# GPU autoscaling nodes.
+# https://github.com/allegroai/clearml-agent/tree/master/docker/k8s-glue
+agentk8sglue:
+  enabled: true
```
Comment (jkhenning) [inline charts/clearml/values.yaml:298]: @nielstenboom I think this should be `false` by default 🙂

**Comment by jkhenning** [comment]
@nielstenboom thanks for this PR!
@valeriano-manassero WDYT?

**Comment by valeriano-manassero** [comment]
> @nielstenboom thanks for this PR!
> @valeriano-manassero WDYT?

Will check it in next 24 hours, ty.

File: charts/clearml/values.yaml (hunk 1 above)
Comment (nielstenboom) [reply charts/clearml/values.yaml]: Ah yes of course, thanks! 👍

**Comment by valeriano-manassero** [comment]
Hi @nielstenboom and ty for the PR, can you pls generate README.md with helm-docs and commit it so the lint and test chart can pass?
When done I will start tests!

**Comment by valeriano-manassero** [comment]
Ty for the PR, new chart version released!

**Comment by nielstenboom** [comment]
Thanks, awesome work on this chart! Made deploying ClearML a piece of cake 😎

**Comment by jkhenning** [comment]
All thanks to @valeriano-manassero 🙏

**Pull request merged**